packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -209,6 +210,14 @@ export class AwsCustomResource extends cdk.Construct implements iam.IGrantable {
       }
     }
 
+    for (const call of [props.onCreate, props.onUpdate, props.onDelete]) {
+      if (call && call.ignoreErrorCodesMatching && call.physicalResourceId?.responsePath) {
+        throw new Error('`PhysicalResourceId.fromResponse` cannot be used along with `ignoreErrorCodesMatching`.');
+      }
+    }
+
+    this.props = props;
```

Comment:
Im not sure about this, I tried looking for a different way to access the resource properties but couldn't find any.
Am i missing something? Are we ok with doing this?

Reply:
It's `private`, so I'm okay with it.